client/src/components/geneExpression/infoPanel/types.ts

```diff
@@ -6,6 +6,12 @@ interface StateProps {
   infoPanelHidden: boolean;
 }
 
+export enum TabsValue {
```

Comment:
should we just drop this enum and use `ActiveTab` enum instead? This way we don't need to do the mapping between String and number!